elastic/apm-server
# Add dynamic_templates to app_metrics data stream

## Motivation/summary

Add a "histogram" dynamic template to the application metrics (`metrics-apm.app-*`) data stream index template, so we can dynamically map histogram metrics when using data streams.

## Checklist

~- [ ] Documentation has been updated~

## How to test these changes

1. Run apm-server with `-E apm-server.data_streams.enabled=true`
2. Send some histogram metrics, e.g. using the Python agent's Prometheus integration, or with OpenTelemetry
3. Check the metrics are indexed as histograms

## Related issues

https://github.com/elastic/apm-server/issues/4636
